bokehjs/src/coffee/core/util/wheel.ts

```diff
@@ -7,7 +7,11 @@
  */
 
 function fontSize(element: Element): number | null {
-  return parseInt(getComputedStyle(element).fontSize, 10) || null
+  const value = getComputedStyle(element).fontSize
+  if (value != null)
+    return parseInt(value, 10)
+  else
+    return null
```

Comment:
Personally I prefer
```
if (value == null)
  return null

return parseInt(value, 10)
```
slightly shorter, doesn't have unneeded branching (when there is a return), and I definitely find comparisons `==` easier to causally reason about then `!=`